algorithmic_efficiency/workloads/ogbg/workload.py

```diff
@@ -17,13 +17,21 @@ class BaseOgbgWorkload(spec.Workload):
 
   _num_outputs: int = 128
 
-  def has_reached_goal(self, eval_result: float) -> bool:
-    return eval_result['validation/mean_average_precision'] > self.target_value
+  def has_reached_validation_target(self, eval_result: float) -> bool:
+    return eval_result[
+        'validation/mean_average_precision'] > self.validation_target_value
 
   @property
-  def target_value(self) -> float:
+  def validation_target_value(self) -> float:
     return 0.28098
 
+  def has_reached_test_target(self, eval_result: float) -> bool:
+    return eval_result['test/mean_average_precision'] < self.test_target_value
```

Comment:
This should be `eval_result['test/mean_average_precision'] > self.test_target_value`.